src/pages/LoginPage.tsx

```diff
@@ -8,13 +8,14 @@ import {
   where,
   query,
 } from 'firebase/firestore';
+import { useDispatch } from 'react-redux';
 import LoginWrapper from '../styles/ReusableStyles';
 import HomeNavBar from '../components/HomeNavBar';
 import { db, auth } from '../firebase/FirebaseTS';
-import UserContext from '../context/Context';
+import { setUser } from '../store/UserSlice';
 
 function LoginPage() {
-  const { setUser }: any = useContext(UserContext);
+  const dispatch = useDispatch();
```

Comment:
Is this working? Since TypeScript, I'd imagine you'd need to useAppDispatch(); (which is also in ./store/hook.ts)